python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -74,16 +74,12 @@ class RandomForestClassifier(BaseRandomForestModel, DelayedPredictionMixin,
         run different models concurrently in different streams by creating
         handles in several streams.
         If it is None, a new one is created.
-    split_criterion : The criterion used to split nodes.
-        0 for GINI, 1 for ENTROPY, 4 for CRITERION_END.
+    split_criterion : int (default = 2)
```

Comment:
Default 2?

Reply:
have fixed it in [this](https://github.com/rapidsai/cuml/pull/4156/commits/a31512ddc8bd6d105e5cf14b0ccb0fcb2ddc9d00) commit rory.